include/wx/toolbook.h

```diff
@@ -2,7 +2,7 @@
 // Name:        wx/toolbook.h
 // Purpose:     wxToolbook: wxToolBar and wxNotebook combination
 // Author:      Julian Smart
-// Modified by:
+// Modified by: Stefan Ziegler
```

Comment:
Just FYI, we've actually stopped using this header since the switch to Git as the list of authors can be easily seen by using `git shortlog` -- which will, of course, show your contributions. So this change is not really useful.